lib/rubocop/cop/style/conditional_assignment.rb

```diff
@@ -436,7 +436,7 @@ def include_ternary?
       # Helper module to provide common methods to ConditionalAssignment
       # correctors
       module ConditionalCorrectorHelper
-        # rubocop:disable Metrics/AbcSize
+        # rubocop:disable Metrics/AbcSize,Metrics/CyclomaticComplexity
```

Comment:
```suggestion
        # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity
```

Reply:
Thank you, I didn't pay attention to this styling detail before. Perhaps it might make sense to a dedicated internal-affairs cop so people don't have to spend time on this